Lab 1/Lab1.ipynb

```diff
@@ -2,11 +2,539 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "7d2b08e5-1d45-4f4e-a574-00ce8587dc23",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.0"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "x+y"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e02a2c2c-25bf-4ebe-9860-49447e74bddb",
+   "metadata": {},
+   "source": [
+    "#### 1C. Getting Z-score based on probabilities."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "aeb2016b-fe16-4b0b-bcdd-e76e8a618d6d",
+   "metadata": {},
+   "source": [
+    "Given a probability of 0.5 (which involves all events from the center of the normal distribution to its left), we predictably get a Z-score of 0."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "0f69b4af-3c53-453f-8fe1-f4c75aaa86df",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "0.0"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(0.5)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "256ac842-8cae-43c9-b5f8-7d094196c532",
+   "metadata": {},
+   "source": [
+    "We know that for a cumulative probability of ~0.84, this corresponds to a distance of +1-sigma away. If we subtract this from 1, we can expect to get a Z-score of roughly -1. This is because we have a probability less than 50 percent, so we know that we will be at a distance of a sigma of negative magnitude."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "dce8389a-d869-4904-9213-fad2a72098b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "-0.9998150936147446"
+      ]
+     },
+     "execution_count": 6,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(1-0.8413)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b33202b4-6982-4eee-a784-4201cdb23684",
+   "metadata": {},
+   "source": [
+    "Here we demonstrate the cdf (cumulative distribution function) and the ppf (percent point function), and how they are inverses of one another."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "fcb54c15-953a-4265-a08c-0b5272205a9e",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "1.0"
+      ]
+     },
+     "execution_count": 7,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "stats.norm.ppf(stats.norm.cdf(1))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "072ee033-5455-4292-ba81-6b896407cafe",
+   "metadata": {},
+   "source": [
+    "#### 2. Exploring other continuous analytic distributions. (Chi-squared)"
```

Comment:
Again, for part 2A give some background on what the Chi-squared distribution is, and when/why it's relevant.

For part 2B, you may want to show multiple plots of the Chi-squared distribution, to show how changing the degrees of freedom parameter changes the plot.